one_fm/api/tasks.py

```diff
@@ -121,17 +130,23 @@ def notify(recipients,log_type):
 	checkout_subject = _("Final Reminder: Please checkout in the next five minutes.")
 	checkout_message = _("""<a class="btn btn-danger" href="/desk#face-recognition">Check Out</a>""")
 	Notification_title = "Final Reminder"
-	user_id = []
-	employee_id = []
+	Notification_body = "Please checkin in the next five minutes."
+	
+	#eg: recipient: ['s.shaikh@armor-services.com', 'HR-EMP-00001', 'Develope-Head Office-Morning-1']
 	for recipient in recipients:

```

Comment:
I think you can use recipient as an object instead of a list with keys: email_id, employee, shift.
That would make more sense.

Reply:
done